### PR TITLE
3rd cycle

### PR DESCRIPTION
**Pull request by chukwuebukaVictor**
This project displays a responsive mobile and desktop version of my portfolio page.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -369,3 +371,188 @@ form {
...
+
+.menu-bar ul {
+  display: none;
+}
+
+.work-bottom2 {
+  display: none;
+}
+
+@media (min-width: 768px) {
```
Comment (SajjadAhmad14) [inline styles.css]: Please use the value for min-width property in range of 992px - 1100px.

File: styles.css [hunk 2]
```diff
@@ -369,3 +371,188 @@ form {
...
+    font-size: 1rem;
+    color: #6070ff;
+    border: 1px solid #dadbff;
+    border-radius: 0.5rem;
+    padding: 0 0.2rem;
+    margin-right: 1rem;
+    background-color: #ebebff;
+  }
+
+  .about-myself,
```
Comment (SajjadAhmad14) [inline styles.css:498]: Please use the same font color for heading, languages, frameworks, and skills as in the design.

File: styles.css [hunk 3]
```diff
@@ -369,3 +371,188 @@ form {
...
+
+  .img-logo {
+    padding: 0 3rem;
+  }
+
+  .menu-bar img {
+    display: none;
+  }
+
+  .intro h1 {
```
Comment (SajjadAhmad14) [inline styles.css:409]: Please use the same color as defined in the design.

File: index.html [hunk 4]
```diff
@@ -35,143 +40,173 @@ <h1>I’m Victor Ozoede Glad to see you!</h1>
...
-          <li class="dev">2015</li>
-        </ul>
-        <p class="body-paragraph">
-          A daily selection of privately personalized reads; no accounts or sign-ups required.
-        </p>
-          <ul class="work-bottom">
-            <li>html</li>
-            <li>css</li>
-            <li>javaScript</li>
+    <section class="work-three work">
```
Comment (SajjadAhmad14) [inline index.html:106]: Font colors for project headings in the work section are also not the same.

File: index.html [hunk 5]
```diff
@@ -35,143 +40,173 @@ <h1>I’m Victor Ozoede Glad to see you!</h1>
...
-        </form>
+          <p class="body-paragraph">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+              <ul class="work-bottom work-bottom1">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+              </ul>
+              <ul class="work-bottom2">
```
Comment (SajjadAhmad14) [inline index.html:151]: The height of the buttons is a little bit large on the tablet view. Please see the design and adjust the height. Kindly see the below screenshot for reference.
buttons

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Victor Ozoede, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
- [ ] GitHub flow is used properly.
- [ ] README is professional.
## To Improve 🙏

- [ ] Please check inline comments under the review
- [ ] Please adjust buttons height on the medium viewport, and follow font colors from design. 
## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 6]
```diff
@@ -35,143 +40,173 @@ <h1>I’m Victor Ozoede Glad to see you!</h1>
...
-        <ul class="script">
-          <li><img src="./img/javascript.svg" alt="js-logo"></li>
-          <li>JavaScript</li>
-        </ul>
-        <ul class="html">
-          <li><img src="./img/html-logo.svg" alt="html-logo"></li>
-          <li>html</li>
+      <div class="flex-item1 flex-item">
+        <h1 class="myself-header">about myself</h1>
+        <p class="myself-paragraph">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me. </p>
```
Comment (Emmyn5600) [inline index.html:167]: - [ ] Optional suggestions

- Work on the `letter-spacing` in your paragraph to get same we have in the figma.

- You can also change the `JavaScript HTML CSS ` seem to be broken.

**Review by Emmyn5600 — APPROVED** [review]
Hi, 
Good job so far! 👏🏼 @chukwuebukaVictor 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed, however use a descriptive feature branch:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**